tests/python/mkl/test_mkldnn.py

```diff
@@ -482,6 +481,10 @@ def check_Deconvolution_training(stype):
             elif np.array(shape).shape[0] == 4:
                 test = mx.symbol.Deconvolution(data=data, kernel=(3, 3), stride=(2, 2), num_filter=4)
                 weight_tmp = np.random.normal(-0.1, 0.1, size=(3, 4, 3, 3))
+            # not testing stype fallback, as 3D deconvolution is not natively supported yet
```

Comment:
Maybe run the test for 3D only when MLDNN is enabled. I think checking compile time "MKLDNN" Features will be enough:
https://mxnet.apache.org/versions/1.8.0/api/python/docs/api/mxnet/runtime/index.html
Additionally you can check if MXNET_MKLDNN_ENABLED environment is not set to false.

Reply:
Aren't the tests from test_mkldnn.py only run with MKLDNN enabled?

Reply:
You are right, so maybe you can change comment to 
```suggestion
            # not testing row_sparse stype, as 3D deconvolution is not natively supported
```

Reply:
Can you check if the comment is clear now?

Reply:
Yes, it is clear